docs/documentation/_data/deckhouse-alerts.yml

```diff
@@ -5834,7 +5834,7 @@ alerts:
 
         Renew the expired certificates if needed.
       summary: |
-        OpenVPN client certificate expired for {{ $labels.client }} in less than 30 days.
+        OpenVPN client certificate expires for {{ $labels.client }} in less than 30 days.
```

Comment:
@maxinsky ... will expire ... ?

Reply:
OpenVPN client certificate {{ $labels.client }} will expire in less than 30 days

?